armi/materials/air.py

```diff
@@ -85,7 +85,7 @@ def pseudoDensity(
         """
         Tk = getTk(Tc, Tk)
         self.checkPropertyTempRange("pseudoDensity", Tk)
-        inv_Tk = 1.0 / getTk(Tc, Tk)
+        inv_Tk = 1.0 / Tk
```

Comment:
This is obviously _better_, but it is does not change the result, right?

`getTk()` will just return `Tk` here. So there is no difference in the mathematical result of this method? (I ask because this says "fix bug" in the title".